MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/annotations/CustomInfoWindowActivity.java

```diff
@@ -43,17 +43,17 @@ public void onMapReady(MapboxMap mapboxMap) {
         mapboxMap.addMarker(new MarkerViewOptions()
           .position(new LatLng(40.416717, -3.703771))
           .anchor(0.5f, 0.5f)
-          .title("spain"));
+          .title("Spain"));
```

Comment:
If/when I do this, the switch case wouldn't work as is because Spain would be translated differently. You can't switch on a non-constant string.

I decided to display flags based on the marker's latitude rather than title. The example still works with my adjustment